library/src/main/java/com/getstream/sdk/chat/viewmodel/ChannelViewModel.java

```diff
@@ -715,7 +716,7 @@ public void onSendMessage(final Message message, @Nullable final MessageCallback
                 callback.onError("no internet", -1);
             return;
         }
-
+        channel.getChannelState().setReadDateOfChannelLastMessage(client().getUser(), message.getCreatedAt());
```

Comment:
why is this here? it looks like some debug code....

Reply:
No debug. After sent message, we must add current user immediately to reads.

Reply:
we do this differently in other sdks, something to be said for this though:

@tbarbugli need your input on how we handle this across SDKs